docs/man/ctags.1.rst

```diff
@@ -686,12 +684,6 @@ Tags File Contents Options
 ``--<LANG>-kinds=[+|-]kinds|*``
 	This option is obsolete. Use ``--kinds-<LANG>=...`` instead.
 
-``--param-<LANG>:name=argument``
```

Comment:
Could you keep ``--param-<LANG>`` as a comment?
Another contributor may think why there is no description for the option.
Keeping this as a comment may help the person understand the situation.